src/org/labkey/test/components/ui/grids/CustomizeGridViewDialog.java

```diff
@@ -324,11 +324,6 @@ public CustomizeGridViewDialog removeColumn(String column)
     public CustomizeGridViewDialog removeColumn(String column, int index)
     {
         WebElement listItem = getShownInGridListItems(column).get(index);
-
```

Comment:
This looks like it should work.
Consider leaving a comment behind to warn future-us about the way repeat-removing columns can appear to leave the cursor over the previous row

Reply:
I think that comment was outdated given that we now move the mouse cursor over the modal title every time.